app/app/settings/docker_dev.py

```diff
@@ -83,5 +83,10 @@
             "level": "INFO",
             "propagate": True,
         },
+        "rq.scheduler": {
+            "handlers": ["console"],
+            "level": "INFO",
+            "propagate": True,
+        },
```

Comment:
This captures all the logs so it's only our logger who writes them.

```suggestion
        "rq_scheduler": {
            "handlers": ["console"],
            "level": "INFO",
            "propagate": False,
        },
        "rq_scheduler.scheduler": {
            "handlers": ["console"],
            "level": "INFO",
            "propagate": False,
        },
```

Reply:
Addressed by: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/403/commits/2beb16f2f1e62f77027debbadd860407e2f3b172